test/integration/smimetrics/smi-metrics_test.go

```diff
@@ -28,6 +28,8 @@ func TestMain(m *testing.M) {
 
 func TestSMIMetrics(t *testing.T) {
 
+	t.Skip("Skipped, as SMI-Metrics currently hardcodes the prometheusUrl of Linkerd which changed")
```

Comment:
Ah nice catch!  We'll need to submit an update to SMI-Metrics to use the Linkerd-viz extension.